<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Test Newtours Site" duration-ms="27659" started-at="2017-08-05T22:00:21Z" finished-at="2017-08-05T22:00:48Z">
    <groups>
    </groups>
    <test name="Complete NewTours Site Test" duration-ms="27659" started-at="2017-08-05T22:00:21Z" finished-at="2017-08-05T22:00:48Z">
      <class name="com.bitm.selenium4thbatch.Test.BookFlightTest">
        <test-method status="FAIL" signature="bookFlightTest(java.util.List)[pri:0, instance:com.bitm.selenium4thbatch.Test.BookFlightTest@1832929]" name="bookFlightTest" duration-ms="0" started-at="2017-08-06T04:00:44Z" data-provider="bookflightdataprovider" finished-at="2017-08-06T04:00:44Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void com.bitm.selenium4thbatch.Test.BookFlightTest.bookFlightTest(java.util.List) throws java.lang.InterruptedException requires a @DataProvider named : bookflightdataprovider in class com.bitm.selenium4thbatch.Test.BookFlightTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void com.bitm.selenium4thbatch.Test.BookFlightTest.bookFlightTest(java.util.List) throws java.lang.InterruptedException requires a @DataProvider named : bookflightdataprovider in class com.bitm.selenium4thbatch.Test.BookFlightTest
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:265)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:414)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bookFlightTest -->
      </class> <!-- com.bitm.selenium4thbatch.Test.BookFlightTest -->
      <class name="com.bitm.selenium4thbatch.Test.LoginTest">
        <test-method status="PASS" signature="checkHomePageTitle()[pri:0, instance:com.bitm.selenium4thbatch.Test.LoginTest@1275dab]" name="checkHomePageTitle" duration-ms="23237" started-at="2017-08-06T04:00:21Z" finished-at="2017-08-06T04:00:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkHomePageTitle -->
        <test-method status="PASS" signature="loginTest(java.util.List)[pri:0, instance:com.bitm.selenium4thbatch.Test.LoginTest@1275dab]" name="loginTest" duration-ms="1418" started-at="2017-08-06T04:00:47Z" data-provider="logindataprovider" finished-at="2017-08-06T04:00:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[LoginDto [username=demo, password=demo]]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- com.bitm.selenium4thbatch.Test.LoginTest -->
      <class name="com.bitm.selenium4thbatch.Test.FlightFinderTest">
        <test-method status="FAIL" signature="flightFinderTest(java.util.List)[pri:0, instance:com.bitm.selenium4thbatch.Test.FlightFinderTest@38203]" name="flightFinderTest" duration-ms="0" started-at="2017-08-06T04:00:48Z" data-provider="flightfinderdataprovider" finished-at="2017-08-06T04:00:48Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.IllegalStateException: Cannot get a STRING value from a NUMERIC cell]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.IllegalStateException: Cannot get a STRING value from a NUMERIC cell
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:143)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.lang.IllegalStateException: Cannot get a STRING value from a NUMERIC cell
	at org.apache.poi.xssf.usermodel.XSSFCell.typeMismatch(XSSFCell.java:1062)
	at org.apache.poi.xssf.usermodel.XSSFCell.getRichStringCellValue(XSSFCell.java:395)
	at org.apache.poi.xssf.usermodel.XSSFCell.getStringCellValue(XSSFCell.java:347)
	at com.bitm.selenium4thbatch.Utils.ExcelUtils.getFlightFinderData(ExcelUtils.java:87)
	at com.bitm.selenium4thbatch.DataProvider.FlightFinderDataProvider.getFlightFinderData(FlightFinderDataProvider.java:14)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:117)
	... 20 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightFinderTest -->
      </class> <!-- com.bitm.selenium4thbatch.Test.FlightFinderTest -->
    </test> <!-- Complete NewTours Site Test -->
  </suite> <!-- Test Newtours Site -->
</testng-results>
